src/edge_containers_cli/cmds/cli.py

```diff
@@ -115,83 +143,89 @@ def deploy(
     """
     Pull an IOC helm chart version from the domain repo and deploy it to the cluster
     """
-    ioc_name = drop_ioc_path(ioc_name)
+    service_name = drop_path(service_name)
     if ctx.obj.namespace == globals.LOCAL_NAMESPACE:
-        IocLocalCommands(ctx.obj, ioc_name).deploy(ioc_name, version, args)
+        IocLocalCommands(ctx.obj, service_name).deploy(service_name, version, args)
     else:
-        IocK8sCommands(ctx.obj, ioc_name).deploy(ioc_name, version, args)
+        IocK8sCommands(ctx.obj, service_name, check=False).deploy(
+            service_name, version, args
+        )
 
 
-@ioc.command()
+@cli.command()
 def list(
     ctx: typer.Context,
```

Comment:
Option to list based on label/s?  --IOC

Reply:
Yes we do want this. But then we probably want to be able to do a full listing with labels shown too - otherwise how to know what labels to use?

Reply:
Will leave this as a todo #99